src/components/tooltip/tooltip.md

```diff
@@ -0,0 +1,59 @@
+The most basic usage of a tooltip (at least for Mavenlink) is typically an icon rendering the tooltip on hover.
+
+```jsx
+import Tooltip from "./tooltip.jsx";
+import Icon from "../icon/icon.jsx";
+import cautionSvg from '../../svgs/caution.svg';
+
+<Tooltip text="and here that beautiful text is" id="help-tooltip" direction="top">
+  <Icon label="caution" icon={cautionSvg} aria-describedby="help-tooltip" />
+</Tooltip>
+```
+
+However, the tooltip is intended to be usable in conjunction with any children or sibling components.
+
+**Regardless, it is _imperative_ that you use `aria-describedby` on the component the tooltip is related to with the `id` provided to `<Tooltip>` to help screenreaders and other assistive technology**.
```

Comment:
```suggestion
**Regardless, it is _imperative_ that there is an `aria-describedby` in the tooltipped component with the value of the `id` provided to `<Tooltip>`. This sets up the accessible relationship which help screenreaders and other assistive technology**.
```

I'm not sure tooltipped is a word. But the original sentence just seemed too packed.